### ISSUE TITLE
Triggers rule when JSX line contains both prop and comment

### ISSUE THREAD

**Issue by razzfox** [yannickcr/eslint-plugin-react#2874]
The rule `jsx-props-no-multi-spaces` triggers when a preceding line contains a comment after a prop.

This code tests for comments: https://github.com/yannickcr/eslint-plugin-react/blob/master/tests/lib/rules/jsx-props-no-multi-spaces.js#L89
The case where a comment exists on the same line as a prop is not tested.

**Current Behavior**

```jsx
<GenericComponent
  prop1='abc' // A comment on this line
  // A comment between props
  prop2='def' // Expected no line gap between “prop1” and “prop2” (react/jsx-props-no-multi-spaces)
```

**Expected Behavior**

```jsx
<GenericComponent
  prop1='abc' // A comment on this line
  // A comment between props
  prop2='def' // OK
```

**Comment by ljharb** [comment]
Should "disabled" and "useful" be "prop1" and "prop2"?

I agree, this shouldn't warn.

**Comment by razzfox** [comment]
Yes that was an edit mistake. The error should show "prop1" and "prop2" in this example. I'll edit to fix that for clarity.
